components/base-node.js

```diff
@@ -44,6 +46,18 @@ module.exports = {
    },
 
   /**
+   * Standard component error handling - send a message to the error port and throw Error
+   *
+   * @param message message to be sent to error port
+   * @param errorPort handle to the component's error port
+   */
+   handleError: function( message, errorPort ) {
```

Comment:
BTW, the location of handleErrors shifted in PR 7.  I have added it to the promiseComponent extension to make it easy for all components to use it.